common/djangoapps/request_cache/middleware.py

```diff
@@ -58,3 +58,51 @@ def process_exception(self, request, exception):  # pylint: disable=unused-argum
         """
         self.clear_request_cache()
         return None
+
+
+def request_cached(f):
```

Comment:
This is a continuation of the discussion of this decorator vs `memoize_in_request_cache in openedx/core/lib/cache_utils.py`.

@tobz mentioned that the main difference between this decorator, and `memoize_in_request_cache`, is that  `memoize_in_request_cache` is required to be used on a class with another method to define the request cache object.

Some questions I have:
1. Could the decorators be combined where the default cache is used if a non-default is not defined?
2. If they must remain separate, which would be preferred?  
3. Is there a way to remove code redundancy between the two, locate them near each other, and/or name them consistently (e.g. `memoize_in_default_request_cache`)?